spec/03-plugins/15-response-transformer/01-header_transformer_spec.lua

```diff
@@ -25,12 +25,11 @@ describe("Plugin: response-transformer", function()
       headers_sent = false,
       resp = {
       },
-      config = {
-        subsystem = "http",
-      },
+      config = ngx.config,
       ctx = {
         KONG_PHASE = 0x00000200,
       },
+      re = ngx.re,
```

Comment:
If re is not set, `jit-uuid` will throw an error because it can't find `ngx.re.find`.